java/memory/memory-netty/src/main/java/io/netty/buffer/PooledByteBufAllocatorL.java

```diff
@@ -166,7 +163,7 @@ private UnsafeDirectLittleEndian newDirectBufferL(int initialCapacity, int maxCa
 
       if (directArena != null) {
 
-        if (initialCapacity > directArena.chunkSize) {
+        if (initialCapacity > chunkSize()) {
```

Comment:
The value of `directArena.chunkSize` is actually equal to `chunkSize()`, and it is a final.

Reply:
Thanks.  One final thing, I think chunkSize is deprecated in Netty? Is it possible to use the non-deprecated version?